packages/components/psammead-image-placeholder/src/index.jsx

```diff
@@ -1,24 +1,30 @@
 import styled from 'styled-components';
-import { number } from 'prop-types';
-import { C_CHALK } from '@bbc/psammead-styles/colours';
-import { BBC_BLOCKS } from '@bbc/psammead-assets/svgs';
+import { number, bool } from 'prop-types';
+import { C_CHALK, C_SHADOW } from '@bbc/psammead-styles/colours';
+import { BBC_BLOCKS, BBC_BLOCKS_DARK_MODE } from '@bbc/psammead-assets/svgs';
 
 const ImagePlaceholder = styled.div`
   position: relative;
   height: 0;
   overflow: hidden;
-  background-color: ${C_CHALK};
+  background-color: ${props => (props.darkMode ? C_SHADOW : C_CHALK)};
   background-position: center center;
   background-repeat: no-repeat;
   background-size: 30%;
   padding-bottom: ${props => props.ratio}%;
   width: 100%;
   /* placeholder BBC blocks SVG */
-  background-image: url(data:image/svg+xml;base64,${BBC_BLOCKS});
+  background-image: url(data:image/svg+xml;base64,${props =>
```

Comment:
This is breaking your build

packages/components/psammead-image-placeholder/src/index.jsx
[App Tests & Code Coverage]  17:3  ✖  Can't parse value "url(data:image/svg+xml;base64,$dummyValue /* dummyComment */)"   csstree/validator

Reply:
ty 👍